dev/RadioButtons/ColumnMajorUniformToLargestGridLayout.cpp

```diff
@@ -133,17 +133,13 @@ void ColumnMajorUniformToLargestGridLayout::OnRowSpacingPropertyChanged(const wi
     InvalidateMeasure();
 }
 
-void ColumnMajorUniformToLargestGridLayout::OnMaximumColumnsPropertyChanged(const winrt::DependencyPropertyChangedEventArgs& args)
+void ColumnMajorUniformToLargestGridLayout::OnMaxColumnsPropertyChanged(const winrt::DependencyPropertyChangedEventArgs& args)
 {
-    InvalidateMeasure();
-}
-
-void ColumnMajorUniformToLargestGridLayout::ValidateGreaterThanZero(int value)
-{
-    if (value <= 0)
```

Comment:
This lost my validation callback changes

Reply:
shoot...